engines/macventure/metaengine.cpp

```diff
@@ -64,7 +64,8 @@ bool MacVentureMetaEngine::hasFeature(MetaEngineFeature f) const {
 		(f == kSavesSupportThumbnail) ||
 		(f == kSavesSupportCreationDate) ||
 		(f == kSimpleSavesNames) ||
-		(f == kSavesSupportPlayTime);
+		(f == kSavesSupportPlayTime) ||
+		(f == kSavesUseExtendedFormat);
```

Comment:
Since you're moving the engine to `kSavesUseExtendedFormat`, the `listSaves()`, `querySaveMetaInfos()` and `removeSaveState()` methods can be removed, since a default implementation exists in `engines/metaengine.cpp`